x/mailbox/keeper/msg_server.go

```diff
@@ -106,14 +104,18 @@ func (k Keeper) Dispatch(goCtx context.Context, msg *types.MsgDispatch) (*types.
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			types.EventTypeDispatch,
-			sdk.NewAttribute(types.AttributeKeySender, msg.Sender),
-			sdk.NewAttribute(types.AttributeKeyDestinationDomain, strconv.FormatUint(uint64(msg.DestinationDomain), 10)),
-			sdk.NewAttribute(types.AttributeKeyRecipientAddress, msg.RecipientAddress),
+			sdk.NewAttribute(types.AttributeKeyDestination, strconv.FormatUint(uint64(msg.DestinationDomain), 10)),
 			sdk.NewAttribute(types.AttributeKeyMessage, msg.MessageBody),
+			sdk.NewAttribute(types.AttributeKeyNonce, strconv.FormatUint(uint64(nonce), 10)),
+			sdk.NewAttribute(types.AttributeKeyOrigin, strconv.FormatUint(uint64(origin), 10)),
+			sdk.NewAttribute(types.AttributeKeyRecipientAddress, msg.RecipientAddress),
+			sdk.NewAttribute(types.AttributeKeySender, msg.Sender),
+			// TODO(nix): How to determine the version? Temporarily set to 0.
+			sdk.NewAttribute(types.AttributeKeyVersion, strconv.FormatUint(0, 10)),
```

Comment:
I'm not sure what this `version` attribute represents. I'm guessing the hyperlane version.